chainio/clients/elcontracts/writer.go

```diff
@@ -72,13 +72,15 @@ func NewELChainWriter(
 
 func BuildELChainWriter(
 	delegationManagerAddr gethcommon.Address,
+	avsContractAddr gethcommon.Address,
```

Comment:
can you rename it to `avsDirectoryAddr`?

Reply:
oops good catch! [e4f61a1](https://github.com/Layr-Labs/eigensdk-go/pull/112/commits/e4f61a16621daec19384a5284d39cec62713a665)